.github/workflows/windows.yml

```diff
@@ -53,8 +53,8 @@ jobs:
         unzip -o "mecab-msvc-x64.zip" -d c:/mecab
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade setuptools wheel pip setuptools-scm build delvewheel
-        pip install -r requirements.txt
+        python -m pip install --upgrade pip setuptools
+        pip install build delvewheel setuptools-scm
```

Comment:
Since you need to call `python -m setuptools_scm`, you need to install `setuptools` and `setuptools-scm`.